pkgs/applications/misc/udiskie/default.nix

```diff
@@ -18,16 +15,27 @@ buildPythonApplication rec {
     sha256 = "1j17z26vy44il2s9zgchvhq280vq8ag64ddi35f35b444wz2azlb";
   };
 
-  buildInputs = [
+  nativeBuildInputs = [
+    gettext
     asciidoc-full        # For building man page.
-    hicolor-icon-theme
+    gobject-introspection
     wrapGAppsHook
+  ];
+
+  buildInputs = [
+    hicolor-icon-theme
     librsvg              # required for loading svg icons (udiskie uses svg icons)
+    gobject-introspection
```

Comment:
```suggestion
```

Reply:
That yields
```
$ udiskie -t &
Typelib for 'libnotify' is not available. Possible causes include:
	- libnotify is not installed
	- the typelib is provided by a separate package
	- libnotify was built with introspection disabled

Starting udiskie without notifications.
Typelib for 'Gtk 3.0' is not available. Possible causes include:
	- GTK3 is not installed
	- the typelib is provided by a separate package
	- GTK3 was built with introspection disabled
Starting udiskie without tray icon.
```
That's why I put it in both.

Reply:
Weird, I would expect setup hooks to work front `nativeBuildInputs` even with `strictDeps`.